internal/provider/resource_attest.go

```diff
@@ -85,14 +92,26 @@ func (r *AttestResource) Schema(ctx context.Context, req resource.SchemaRequest,
 				MarkdownDescription: "This always matches the input digest, but is a convenience for composition.",
 				Computed:            true,
 			},
+			"fulcio_url": schema.StringAttribute{
+				MarkdownDescription: "Address of sigstore PKI server (default https://fulcio.sigstore.dev).",
+				Optional:            true,
+				Computed:            true,
```

Comment:
If i remove that it complains in the go generate

```
$ go generate ./...
rendering website for provider "terraform-provider-cosign" (as "terraform-provider-cosign")
exporting schema from Terraform
compiling provider "cosign"
using Terraform CLI binary from PATH if available, otherwise downloading latest Terraform CLI binary
running terraform init
getting provider schema
Error executing command: unable to generate website: exit status 1

Error: Failed to load plugin schemas

Error while loading schemas for plugin components: Failed to obtain provider
schema: Could not load the schema for provider
registry.terraform.io/hashicorp/cosign: failed to retrieve schema from
provider "registry.terraform.io/hashicorp/cosign": 2 problems:

- Schema Using Attribute Default For Non-Computed Attribute: Attribute
"fulcio_url" must be computed when using default. This is an issue with the
provider and should be reported to the provider developers.
- Schema Using Attribute Default For Non-Computed Attribute: Attribute
"rekor_url" must be computed when using default. This is an issue with the
provider and should be reported to the provider developers...
```